controllers/common-controller-utils.go

```diff
@@ -44,10 +44,10 @@ func createOrUpdateDestinationSecretsFromSource(ctx context.Context, rc client.C
 
 	uniqueConnectedPeers, err := PeersConnectedToSecret(sourceSecret, mirrorPeers)
 	if err != nil {
-		logger.Error(err, "ConnectedPeers returned an error", "secret", sourceSecret, "mirrorpeers", mirrorPeers)
+		logger.Error(err, "ConnectedPeers returned an error", "secret", sourceSecret.Name, "mirrorpeers", mirrorPeers)
 		return err
 	}
-	logger.V(2).Info("Listing all the Peers connected to the Source", "SourceSecret", sourceSecret, "#connected-peers", len(uniqueConnectedPeers))
+	logger.V(2).Info("Listing all the Peers connected to the Source", "SourceSecret", sourceSecret.Name, "#connected-peers", len(uniqueConnectedPeers))
 
 	// anyErr will have the last found error
```

Comment:
Line 58 in the file. We are logging the peerref instead of the secret name. 
Please change that